File: web-service/security/src/main/java/io/ehdev/conrad/security/SignInController.java
package io.ehdev.conrad.security;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

@Controller
public class SignInController {

    @RequestMapping(value = "/signin", method = RequestMethod.GET)
    public String getSignInPage() {
        return "signin";
    }

    @RequestMapping("/signup")
    public void getSignUpPage() { }
}

File: web-service/webapp/src/main/kotlin/tech/crom/webapp/endpoint/SigninController.kt
package tech.crom.webapp.endpoint

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RequestMethod

@Controller
class SigninController {

    @RequestMapping(value = "/signin", method = arrayOf(RequestMethod.GET))
    fun signin(): String = "signin"
}